sonar-plugin-api/src/main/java/org/sonar/api/test/MutableTestPlan.java

```diff
@@ -19,15 +19,13 @@
  */
 package org.sonar.api.test;
 
-import org.sonar.api.component.MutablePerspective;
-
-public interface MutableTestPlan extends TestPlan<MutableTestCase>, MutablePerspective {
+public interface MutableTestPlan extends TestPlan<MutableTestCase> {
```

Comment:
![New](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/new.png)![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png)"TestPlan" is deprecated, implement the suggested replacement instead.[![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1724)